lib/internal/util/inspect.js

```diff
@@ -1283,6 +1283,51 @@ function removeDuplicateErrorKeys(ctx, keys, err, stack) {
   }
 }
 
+function markNodeModules(ctx, line) {
+  let tempLine = '';
+  let nodeModule;
+  let pos = 0;
+  while ((nodeModule = nodeModulesRegExp.exec(line)) !== null) {
+    // '/node_modules/'.length === 14
+    tempLine += line.slice(pos, nodeModule.index + 14);
```

Comment:
ArrayPrototypeSlice

Reply:
There is no agreement about using primordials everywhere in the codebase so far, just for error cases.

Reply:
That’s not my understanding of the TSC vote. Either way, inspection is done often in error paths, and so this code qualifies.

Reply:
That would be `StringPrototypeSlice` I think, not `ArrayPrototypeSlice`. But anyway, since this is not strictly on the error path, primordials are opt-in, not using them is not a sufficient reason for blocking that PR – refactor to primordials could be done in a follow-up PR.

Reply:
I can’t block a PR in any case. I’m requesting they be done so that this robust code path doesn’t become brittle.